index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+     <script src="https://kit.fontawesome.com/8c1870b6fa.js" crossorigin="anonymous"></script>
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <!--header section start-->
+    <header>
+      <h2 class="logo"><a href="#">Fatima</a></h2>
+      <nav>
+        <div class="hamburger">
+          <svg width="750px" height="750px" viewBox="0 0 32 32" xmlns="http://www.w3.org/2000/svg" fill="none">
+            <path stroke="#6070FF" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M5 8h22M5 16h22M5 24h22"/>
+          </svg>
+        </div>
+       
+      </nav>
+    </header>
+    <!--header section ends-->
+
+    <!--headline section starts-->
+    <section class="headline-main">
+      <section class="headline">
+        <h2 class="headline-heading">I'm Fatima.<br />Glad to see you!</h2>
```

Comment:
- [ ] Hi, great job on your web page. I noticed that you used an h2 tag for the main title of your web page (e.g., “I’m Marianne …”). I suggest that you change it to an h1 tag instead. The reason is that an h1 tag is the most important heading on your web page and it tells Google (and other search engines) what your web page is about. In this case, you want Google to know that your web page is about you. Using an h1 tag will also improve the accessibility and readability of your web page for users who use screen readers or have low vision. You can check this link for more info: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements. Please make this change and resubmit your project. Thank you for your cooperation. 😊